src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DomainUpDownTests.cs

```diff
@@ -7,273 +7,278 @@
 
 namespace System.Windows.Forms.Tests;
 
-public class DomainUpDownTests
+public class DomainUpDownTests : IDisposable
 {
+    private readonly DomainUpDown _control;
+    private readonly SubDomainUpDown _sub;
+
+    public DomainUpDownTests()
+    {
+        _control = new DomainUpDown();
```

Comment:
```suggestion
        _control = new();
```

We prefer "target-typed new" syntax